high_card.rb

```diff
@@ -3,3 +3,89 @@
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+deck = ranks.product(suits).shuffle!
+
+class Player
+    attr_reader :card, :name
+    @@player_count = []
+    def initialize player_name
+        @name = player_name
+        @card = []
+        @@player_count.push({name: "#{@name}"})
+    end
+
+    def self.total_players
+        "#{@@player_count}"
+    end
+
+    def deal_cards deck
+        @card = deck.sample
+    end
+end
+
+class Game < Player
```

Comment:
I'm not sure I would have Game inherit from Player. A Game would have a player name...